.github/workflows/linux-build.yml

```diff
@@ -66,6 +66,9 @@ jobs:
           skip-go-installation: true
           skip-pkg-cache: true
 
+      - name: Nancy Security scanner
+        run: go list -json -deps | docker run --rm -i sonatypecommunity/nancy:latest sleuth
+
```

Comment:
maybe we should also add a dependncy check wokflow here, that checks dependencies each night.

Reply:
✅